src/components/ChainConnection.tsx

```diff
@@ -116,7 +132,7 @@ const ChainConnection = () => {
           'border border-primary group inline-flex items-center rounded-md px-3 py-2 bg-transparent text-base font-medium text-primary',
           !connectionInProgress && !chainConnection && 'hover:bg-gray-100'
         )}
-        onClick={connect}
+        onClick={() => connect()}
```

Comment:
what happens without this change?

Reply:
It's just a type signature problem, `onClick` is an event handler and I gave `connect` a param which doesn't follow that interface